code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -760,7 +760,7 @@
 		visible_message("<span class='notice'>[src] safeties are locked on.</span>")
 
 /mob/living/simple_animal/bot/mulebot/paranormal//allows ghosts only unless hacked to actually be useful
-	name = "paranormal MULEbot"
+	name = "\improper GHOULbot"
 	desc = "A Multiple Utility Load Effector bot. It only seems to accept paranormal forces, and for this reason is fucking useless."
```

Comment:
it still says Multiple Utility Load Effector bot in the desc

Reply:
intended. feel free to suggest a change though!

Reply:
Ghastly Hauling Of UnLife bot? Eh, it's a bit of a stretch, and do ss13 ghosts really count as unlife, seeing as how they're more "dead" than "undead"?